packages/editor/src/components/blocks/text/Text.tsx

```diff
@@ -30,9 +33,22 @@ export const TextComponent: ComponentConfig<TextProps> = {
   fields: {
     content: { subsection: 'General', label: 'Content', type: 'textarea' },
     type: { subsection: 'General', label: 'Type', type: 'select', options: typeOptions },
+    icon: { subsection: 'General', label: 'Icon', type: 'text' },
     ...baseComponentFields
   },
   quickActions: DEFAULT_QUICK_ACTIONS
 };
 
-const UiBlock = ({ content }: UiComponentProps<TextProps>) => <p className='block-text'>{content}</p>;
+const UiBlock = ({ content, icon }: UiComponentProps<TextProps>) => {
+  if (icon) {
+    console.log(icon);
+    return (
+      <div className='text-with-icon'>
+        <IvyIcon icon={IvyIcons.InfoCircle} />
+        <p className='block-text'>{content}</p>
+      </div>
+    );
+  } else {
+    return <p className='block-text'>{content}</p>;
```

Comment:
why not `<p className='block-text'>{icon && <IvyIcon icon={icon} />}{content}</p>`?